### PR TITLE
Progress satisfying the first user story

### PR DESCRIPTION
**Pull request by tobenna**

### CODE REVIEW

**Comment by mechacoach** [comment]
You had your airport_challenge reviewed by **Chris Wynne**

### The good points are:

* Overview of project in the README
* `context` or `describe` blocks are used (where appropriate) to group tests
* The solution avoids vacuous tests
* `before` blocks are used where appropriate
* There are a minimal number of untested paths in the code
* Unit tests all have single expectations
* Random behaviour is stubbed where appropriate
* The code follows Ruby naming conventions, Readability is supported by consistent naming
* Guard clauses are used where appropriate, Booleans are returned implicitly where appropriate, removeallcommented-outcode: There is no commented-out code in the submission
* Code avoids unnecessary public methods, Code avoids public access to internal objects (e.g. planes array)
* Classes have single responsibilities, Each methods has a single responsibility
* Code avoids strings where it should use symbols
* Command/Query naming is not an issue in the code
* The solution does not have any redundant code

### You should consider the following improvements:

* Installation Instructions should be in the README
* Use named subject with `described_class`
* Get rid of redundant `respond_to` expectations
* Remove all commented-out code
* Avoid magic numbers (e.g. on capacity)
* Improper use of attr_*

### Additional comments:
Good stuff. Bit of research on use of attr would be good, and the use of context blocks to lay out your tests would be great. Overall really good though., avoidmagicnumberse.g.oncapacity: Code avoids magic numbers

see https://github.com/makersacademy/airport_challenge/blob/master/docs/review.md for more details

**Comment by silvabox** [comment]
Hi Tobenna, this is a pretty good effort - well done.  I've made a couple of comments above.

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  def remove_from_airfield(plane)
+    @airfield.delete_if { |pl| pl == plane }
+  end
+
+  def at_capacity?
+    capacity == airfield.size
+  end
+
+  def can_land?
```
Comment (silvabox) [inline lib/airport.rb]: by convention, a method with a `?` should return true or false; not true or raise an error.

File: lib/weather.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  def stormy?
+    forecast == :stormy
+  end
+
+  private
+
+  WEATHER_SEED = { sunny: 19, stormy: 1 }.freeze
+
+  def weather_array
+    weather = []
```
Comment (silvabox) [inline lib/weather.rb]: Is it necessary to recreate this array on every call to `weather_array`?

**Pull request closed without merging**